wrapanapi/systems/scvmm.py

```diff
@@ -153,7 +256,7 @@ def all_ips(self):
     def creation_time(self):
         self.refresh()
         creation_time = convert_powershell_date(self.raw['CreationTime'])
-        return creation_time.replace(tzinfo=tzlocal.get_localzone()).astimezone(pytz.UTC)
+        return creation_time.replace(tzinfo=self.system.timezone).astimezone(pytz.UTC)
```

Comment:
So this slightly modifies the behavior if the TZ is not understood in the called property.

In that case we'll be setting `tzinfo=None` - I think its good to have the property return None if its not understood - but here we should use `tzinfo=self.system.timezone or tzlocal.get_localzone()`.

Reply:
Agreed, I'll update this